Dockerfile

```diff
@@ -64,6 +77,8 @@ ARG GRAYLOG_UID=1100
 ARG GRAYLOG_GROUP=graylog
 ARG GRAYLOG_GID=1100
 
+COPY --from=graylog-downloader /opt/tini /tini
```

Comment:
Should this not also COPY `tini` into `/opt`, to follow the same pattern as the `/opt/graylog` install?

Reply:
good call-out :ok_hand:; I thought about this end ended up intentionally making `/tini` the location in the final layer because that's the location [in the Tini docs](https://github.com/krallin/tini#using-tini) and also the one I see most often in the wild.

I use `/opt/tini` in the downloader layer to conform with the existing convention of using `/opt/` for downloaded executables.

I'm 100% open to any location and happy to update to accommodate --just took my best shot in the dark to remain idiomatic :smile:

good feedback!